mender-convert

```diff
@@ -33,7 +33,7 @@ Expert commands:
                                                   related files
 
 Options: [-r|--raw-disk-image | -m|--mender-disk-image | -s|--data-part-size-mb |
-          -d|--device-type | -p|--rootfs-partition-id | -i|--demo-host-ip |
+          -d|--device-type | -p|--rootfs-partition-id | -e|--demo | -i|--demo-host-ip |
```

Comment:
Conflict between '--demo' and '--storage-total-size-mb' - both use short name as '-e'

Reply:
Good catch, thank you.